tools/idl_interfaces.sh

```diff
@@ -54,7 +59,7 @@ function generate_subcore_interface {
     # This option will be the name of the imported header and the correponding variable (defined
     # by EXPORT_MACRO_NAME option) should be exactly same name in upper case
 
-    ./djinni/src/run \
+    ../djinni/src/run \
```

Comment:
`HODL` until https://github.com/LedgerHQ/djinni/pull/43 is merged and this is reverted to `./…`.